test/functional/feature_staking_unlock_and_withdraw.py

```diff
@@ -87,9 +88,16 @@ def run_test(self):
         assert_equal(events[2].value['event_id'], 'StakeUnlocked')
 
         ledger = list(client.get_staking_ledger())
+
         assert_equal(len(ledger),2)
 
-        time.sleep(500)
+        num_attempts = 0
+        max_attempts = 10000
```

Comment:
10000 seconds is like 3 hours... you sure?

Reply:
@TheQuantumPhysicist I chose this a bit randomly.... Should we go for 20 minutes? This is twice as long as it *should* take:

Reply:
Well, I'd prefer that you change this whole thing to wait for time units, not retry count. If it isn't clear what I mean, hit me with a DM.